controls/SV-204597.rb

```diff
@@ -1,25 +1,28 @@
 control 'SV-204597' do
-  title 'The Red Hat Enterprise Linux operating system must be configured so that the SSH private host key files
-    have mode 0600 or less permissive.'
+  title 'The Red Hat Enterprise Linux operating system must be configured so that the SSH private host key files have mode 0640 or less permissive.'
```

Comment:
```yml
  # SV-204597
  - name: private_host_key_file_mode
    description: Maximum mode allowed for directories containing SSH private host keys (in octal format)
    type: String
    value: "0640"
```

Why are we providing an option to the users to modify this value? This could lead to non-compliance.